Consul.Test/ConnectTest.cs

```diff
@@ -208,5 +208,50 @@ public async Task Connect_UpsertIntentionByName()
             Assert.True(testIntention.Precedence > 0);
             await _client.Configuration.DeleteConfig("service-intentions", newEntry.DestinationName);
         }
+
+        [SkippableFact]
+        public async Task Connect_ReadSpecificIntentionByName()
+        {
+            var cutOffVersion = SemanticVersion.Parse("1.9.0");
+            Skip.If(AgentVersion < cutOffVersion, $"Current version is {AgentVersion}, but `service intentions` are only supported from Consul {cutOffVersion}");
+
+            var newEntry = new ServiceIntentionsEntry
+            {
+                Kind = "service-intentions",
+                Name = "Oscars",
+                Sources = new List<SourceIntention>
+                {
+                    new SourceIntention
+                    {
+                        Name = "Kanye West",
+                        Action = "deny",
+                        LegacyCreateTime = DateTime.UtcNow,
+                        LegacyUpdateTime = DateTime.UtcNow,
+                    }
+                }
+            };
+
+            var source = newEntry.Sources.First(x => x.Name == "Kanye West").Name;
```

Comment:
Please define the source name and destination name as variables, that will let you get rid of this `Sources.First(x => x.Name == "Kanye West").Name` which is clunky.